### DISCUSSION TITLE
My admin acc

### DISCUSSION THREAD

**Discussion by prettychicagoboy1** [community/community#111202] [Enterprise]
I am just looking to verify that my domain www.jjlmedia.com has been updated to reflect my name ownership of the domain as I'm the CEO of JJL Media and was hacked and had lost admin access. My email address should also be jamesloughman@jjlmedia.com

I also am the owner of jjlmediasolutions@gmail.com and Prettychicagoboy@gmail.com

**Comment by ossanna16** [comment]
Hi @prettychicagoboy1 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
